justfile

```diff
@@ -1,4 +1,5 @@
-export RUST_LOG := "wascc_host=debug,wascc_provider=debug,wasi_provider=debug,main=debug,kubelet=debug"
+export RUST_LOG := "wascc_host=debug,wascc_provider=debug,wasi_provider=debug,main=debug"
```

Comment:
Why did you remove the `kubelet` log level? too noisy?

Reply:
> Also, I removed the kubelet=debug setting for logging because kubelet is
a really verbose default